src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,397 @@
+   * used to allow or deny all values. If allValues is set to either ALLOW or DENY, allowedValues
+   * and deniedValues must be unset.
+   */
+  public static class ListPolicy implements Serializable {
+    private static final long serialVersionUID = -2133042982786959352L;
+
+    private final String allValues;
+    private final List<String> allowedValues;
+    private final List<java.lang.String> deniedValues;
+    private final Boolean inheritFromParent;
+    private final String suggestedValue;
+
+    public ListPolicy(
+        String allValues,
+        List<String> allowedValues,
+        List<String> deniedValues,
+        Boolean inheritFromParent,
+        String suggestedValue) {
+      this.allValues = allValues;
+      this.allowedValues = allowedValues;
+      this.deniedValues = deniedValues;
+      this.inheritFromParent = inheritFromParent;
+      this.suggestedValue = suggestedValue;
+    }
+
+    public String getAllValues() {
+      return allValues;
+    }
+
+    public List<String> getAllowedValues() {
+      return allowedValues;
+    }
+
+    public List<String> getDeniedValues() {
+      return deniedValues;
+    }
+
+    public Boolean getInheritFromParent() {
+      return inheritFromParent;
+    }
+
+    public String getSuggestedValue() {
+      return suggestedValue;
+    }
+
+    @Override
+    public String toString() {
+      return MoreObjects.toStringHelper(this)
+          .add("allValues", getAllValues())
+          .add("allowedValues", getAllowedValues())
+          .add("deniedValues", getDeniedValues())
+          .add("inheritFromParent", getInheritFromParent())
+          .add("suggestedValue", getSuggestedValue())
+          .toString();
+    }
+
+    @Override
+    public final boolean equals(Object obj) {
+      return obj == this
+          || obj != null
+              && obj.getClass().equals(ListPolicy.class)
+              && Objects.equals(toPb(), ((ListPolicy) obj).toPb());
+    }
+
+    @Override
+    public int hashCode() {
+      return Objects.hash(
+          allowedValues, allowedValues, deniedValues, inheritFromParent, suggestedValue);
+    }
+
+    com.google.api.services.cloudresourcemanager.model.ListPolicy toPb() {
+      return new com.google.api.services.cloudresourcemanager.model.ListPolicy()
+          .setAllValues(allValues)
+          .setAllowedValues(allowedValues)
+          .setDeniedValues(deniedValues)
+          .setInheritFromParent(inheritFromParent)
+          .setSuggestedValue(suggestedValue);
+    }
+
+    static ListPolicy fromPb(
+        com.google.api.services.cloudresourcemanager.model.ListPolicy listPolicy) {
+      return new ListPolicy(
+          listPolicy.getAllValues(),
+          listPolicy.getAllowedValues(),
+          listPolicy.getDeniedValues(),
+          listPolicy.getInheritFromParent(),
+          listPolicy.getSuggestedValue());
+    }
+  }
+
+  public abstract static class Builder {
```

Comment:
why the abstract class?

Reply:
Already removed abstract class and used concrete Builder class